textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriterTest.java

```diff
@@ -0,0 +1,114 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import edu.uci.ics.textdb.api.common.ITuple;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ * Testing QueryRewriter
+ * This test class tests the functionality of the QueryRewrite operator
+ * this is done by feeding the class with different kinds of strings to
+ * try the query rewriting.
+ *
+ * This includes query strings like - "newyorkcity", "horseshoe" and other
+ * strings which stem from misspelled, missing spaces in search queries
+ */
+public class QueryRewriterTest {
+
+    /**
+     * Tests the QueryRewriter operator with the string "horseshoe"
+     * @throws Exception
+     */
+    @Test
+    public void testHorseShoeString() throws Exception {
+        String query = "horseshoe";
+
+        QueryRewriter queryRewriter = new QueryRewriter(query);
+        queryRewriter.open();
+        ITuple iTuple = queryRewriter.getNextTuple();
+        queryRewriter.close();
+
+        Assert.assertNotNull(iTuple);
+
+        List<String> fuzzySet = (List<String>) iTuple.getField(0).getValue();
+        Assert.assertNotNull(fuzzySet);
+
+        List<String> correctSet = Arrays.asList("hor se shoe",
+                "hors es hoe",
+                "horse shoe",
+                "horses hoe",
+                "horseshoe");
+        Collections.sort(correctSet);
+        Collections.sort(fuzzySet);
+
+        Assert.assertEquals(1,1);
```

Comment:
@kishore-narendran : in your current test case, the computed set "fuzzySet()" is not really computed yet since the operator is not implemented.  If you agree, you may not use containsAllResults() in this PR.  In other words, your test case is not "complete" yet.

In the next PR with the operator implementation, you should complete the test case and use containsAllResults().  

Agreed?